apps/voting/app/src/components/SummaryRows.js

```diff
@@ -11,7 +12,7 @@ const SummaryRows = ({ yea, nay, symbol }) => (
         <div>{yea.pct}%</div>
       </RowStart>
       <Amount>
-        {yea.amount} {symbol}
+        {formatNumber(yea.amount, 5)} {symbol}
```

Comment:
Right now this always forces 5 decimal points. Given that most people will be using whole units for tokens, at least initially, this could look odd (that all the decimals are always `.00000`):

<img src="https://user-images.githubusercontent.com/4166642/46900364-2de77480-cea1-11e8-9340-5d1d3b792254.png" height=200>

------------

Wonder if we should not display the decimals if it's a whole number?

Reply:
Yes maybe we could check that both are whole numbers and remove the decimals if that’s the case?

Reply:
We should also probably adapt it based `tokenDecimals` if less than 5.